docs/topics/sealed-classes.md

```diff
@@ -12,16 +12,23 @@ In some sense, sealed classes are similar to [`enum`](enum-classes.md) classes:
 for an enum type is also restricted, but each enum constant exists only as a _single instance_, whereas a subclass
 of a sealed class can have _multiple_ instances, each with its own state.
 
+As an example, consider a hierarchy of possible errors in a library. There can be classes for networking, database, and
```

Comment:
It's a bit confusing: classes for ..., ..., and other errors — these entities don't have the same meaning, IDK can we enumerate them in that way?